src/lib/badges/Badge.svelte

```diff
@@ -31,36 +29,70 @@
     ['!pink']: 'bg-pink-500 text-pink-100'
   };
 
-  const hovers = {
-    blue: 'hover:bg-blue-200 dark:hover:bg-blue-300',
-    dark: 'hover:bg-gray-200 dark:hover:bg-gray-300',
-    red: 'hover:bg-red-200 dark:hover:bg-red-300',
-    green: 'hover:bg-green-200 dark:hover:bg-green-300',
-    yellow: 'hover:bg-yellow-200 dark:hover:bg-yellow-300',
-    indigo: 'hover:bg-indigo-200 dark:hover:bg-indigo-300',
-    purple: 'hover:bg-purple-200 dark:hover:bg-purple-300',
-    pink: 'hover:bg-pink-200 dark:hover:bg-pink-300'
```

Comment:
The hover styles in dark mode are not set in flowbite's official examples, so I directly delete previous `dark:hover:` styles, maybe they can be re-added after flowbite makes some official `dark:hover:` style suggestions.